docs/ConfigureScalarDL.md

```diff
@@ -2,126 +2,43 @@
 
-```yaml
-ledger:
-  ....
-  scalarLedgerConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbPassword: <Cosmos DB account primary master key>
-    dbStorage: cosmos
-```
+| Database  | storageType | contactPoints              | username       | password                                 | dynamoBaseResourceUnit | cosmosBaseResourceUnit | 
+|:----------|-------------|----------------------------|----------------|------------------------------------------|------------------------|------------------------|
+| DynamoDB  | dynamo      | AWS_REGION                 | AWS_ACCESS_KEY | AWS_ACCESS_SECRET_KEY                    | 10                     | N/A                    |
+| Cosmos DB | cosmos      | Cosmos DB account endpoint | N/A            | Cosmos DB account primary/secondary key  | N/A                    | 400                    |
+| JDBC      | jdbc        | JDBC_CONNECTION_URL        | USERNAME       | PASSWORD                                 | N/A                    | N/A                    |
 
-To enable Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
+Note:
 
-#### Configure scalardl-audit-custom-values
+* JDBC denotes all relational databases supported by Scalar DL such as MySQL, PostgreSQL, Oracle Database, SQL Server, and Amazon Aurora.
 
-To deploy the Scalar DL Auditor on Cosmos DB, you need to update the [scalardl-audit-custom-values.yaml](../conf/scalardl-audit-custom-values.yaml) file.
+## Create Kubernetes Secrets
 
-**With Kubernetes Secrets**
+Kubernetes Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key.
+This optional method is recommended highly for handling credentials in the production environment.
+You can refer to the [reference table](#reference-table) section to get the values for `username` and `password` for the database you chose.
+You need to create a secret object with `db-username` and `db-password` to store the username and password of the underlying database.
 
-If you have created the `scalardl` Kubernetes secret, you can use the following configuration.
 
-```yaml
-auditor:
-  existingSecret: scalardl
-  ....
-  scalarAuditorConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbStorage: cosmos
 ```
-
-**Without Kubernetes secrets**
-
-If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.
-
-```yaml
-auditor:
-  ....
-  scalarAuditorConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbPassword: <Cosmos DB account primary master key>
-    dbStorage: cosmos
+kubectl create secret generic scalardl --from-literal db-username=<username> --from-literal db-password=<password>
 ```
+Note:
 
-To enable the Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
+For Cosmos DB, use the Cosmos DB account name as `username`.
```

Comment:
If we use Cosmos DB, I think the value of `username` is not used to access it.
Is my understanding correct?

If yes, I think it is not good to include `account name` (credential information) that is not actually used from the perspective of security.
The secret resource is designed to be secure, but some user (e.g. administrator) can read the value of secret resource.

I think it is better to explain the above (`username` is not actually used in Cosmos DB) and use some dummy strings (e.g. `dummy-user`) instead of `account name`.